### ISSUE TITLE
[cPP PUBLIC REVIEW] FCS_STG_EXT.1.1 mismatch

### ISSUE THREAD

**Issue by jvdsn** [DSC-iTC/cPP#386]
FCS_STG_EXT.1.1 as defined in the ECD does not match FCS_STG_EXT.1.1 as instantiated in the cPP.

ECD definition:

FCS_STG_EXT.1.1:: The TSF shall provide [assignment: _protection method_] protected storage for asymmetric private keys and [selection: _symmetric keys, persistent secrets, no other keys_].

cPP instantiation:

FCS_STG_EXT.1.1:: The TSF shall provide {empty}[selection: [.underline]#mutable hardware-based, immutable hardware-based, software-based in accordance to FCS_CKM_EXT.3]# protected storage for {empty}[selection: [.underline]#asymmetric private keys, symmetric keys]# and {empty}[selection: [.underline]#persistent secrets, no other keys]#.

(Ignore the formatting for now).

In the ECD, asymmetric private keys is not part of the selection, there is only one selection. In the cPP instantiation there's two selections and asymmetric private keys is part of one of them. I'm inclined to say the cPP instantiation is correct.

**Comment by jvdsn** [comment]
I'm also not sure why asymmetric keys is actually "mandatory". Is the idea that a DSC will always have at least one asymmetric key? I don't think we can change the actual meaning of the SFR now, so the least invasive option is to change the ECD to match the cPP body.

**Comment by jvdsn** [comment]
Maybe it should just be one big selection that includes asymmetric private, symmetric, persistent secrets, and no other keys

**Comment by woodbe** [comment]
The initial discussion was by BSI wanting to be able to do symmetric keys as mandatory, but we ended up not being able to come up with any actual use case where only symmetric keys would be stored, but we moved symmetric to mandatory.

Take a look at the pull request. Basically instead of having the weird first selection I remove it to just make them mandatory and then leave in the one at the end.

Also removes the underlined part because that was an assignment moving to a selection, so it shouldn't be there.
